avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -165,6 +166,37 @@ public void testFromConnectString() {
     checkNonRecordConversionNull(Schema.OPTIONAL_STRING_SCHEMA);
   }
 
+  @Test
+  public void testFromConnectBytesFixed() {
+    org.apache.avro.Schema avroSchema = org.apache.avro.SchemaBuilder.builder().fixed("sample").size(4);
+    GenericData.Fixed avroObj = new GenericData.Fixed(avroSchema, "foob".getBytes());
+    avroSchema.addProp("connect.parameters", ImmutableMap.of("connect.fixed.size", "4"));
+    avroSchema.addProp("connect.name", "sample");
+    SchemaAndValue schemaAndValue = avroData.toConnectData(avroSchema, avroObj);
+    checkNonRecordConversion(avroSchema, avroObj, schemaAndValue.schema(), schemaAndValue.value(),
+      avroData);
+  }
+
+  @Test
+  public void testFromConnectFixedUnion() {
+    org.apache.avro.Schema unionSchema = org.apache.avro.SchemaBuilder.builder().unionOf()
+      .type(avroFixed("sample", 4)).and()
```

Comment:
add a third branch of the union, another fixed with a different name but same size as one of the above.  It should be able to match the right one even if the size is the same.

Reply:
Done.